src/coreComponents/constitutive/fluid/multifluid/CO2Brine/CO2BrineFluid.cpp

```diff
@@ -177,7 +179,8 @@ void CO2BrineFluid< PHASE1, PHASE2, FLASH >::checkTablesParameters( real64 const
     m_flash->checkTablesParameters( pressure, temperatureInCelsius );
   } catch( SimulationError const & ex )
   {
-    string const errorMsg = GEOS_FMT( "{}: Table input error for flash phase.\n", getDataContext() );
+    string const errorMsg = GEOS_FMT( "Table input error for flash phase (in table from \"{}\").\n",
```

Comment:
why are you removing `getDataContext` here and also above?

Reply:
Because this exception is stacked with another one that show a more useful `DataContext`, I'm removing this one because it is reducing clarity for the users.